daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -70,6 +70,22 @@ spec:
     enabled: false
 ```
 
+## Configuring metrics for error codes
+
+You can enable additional metrics for [Dapr API error codes](https://github.com/dapr/dapr/blob/master/docs/reference/api/error_codes/) by setting `spec.metrics.recordErrorCodes` to `true`. See the specific metrics described in the [Dapr development docs](https://github.com/dapr/dapr/blob/master/docs/development/dapr-metrics.md).
```

Comment:
I think this section should answer:
- why do I want to enable this?
- what happens when I enable this?

Also, mention metrics are collected and maybe add some metric examples

Reply:
@jake-engelberg friendly bump